### PR TITLE
Add power signal; somewhat limited

### PR DESCRIPTION
**Pull request by elffjs**
For Teslas, we have this field `charger.power`:
```
{
  …
  "data": {
    …
    "charger": {
      "power": 28
    }
  }
}
```
In kilowatts. This is only populated during charging. [Looking at VSS, the best match seems to be `Powertrain.TractionBattery.CurrentPower`](https://github.com/COVESA/vehicle_signal_specification/blob/86891874413364376c7ac66617559bacafc95dc7/spec/Powertrain/TractionBattery.vspec#L186). Two weird things:

* It's _supposed_ to be populated all the time, with negative values when the vehicle is consuming power while driving. We don't have those drive consumption values. We can look and see if we can get it—this won't help with historical payloads.
* It's in watts, so we must multiply the historical stuff by 1000.

### CODE REVIEW

**Comment by elffjs** [comment]
I think one does want to add `-package vss` there. Same error though.

**Comment by elffjs** [comment]
I see, it's because `all` implies `custom`. This is if you want to execute some non-standard template?

It seems like `go run github.com/DIMO-Network/model-garage/cmd/codegen -output=pkg/vss -package vss -generators=model,convert` is good enough, then.

File: pkg/vss/vehicle-structs.go [hunk 1]
```diff
@@ -58,6 +58,8 @@ const (
 	FieldPowertrainTractionBatteryChargingChargeLimit = "powertrainTractionBatteryChargingChargeLimit"
 	// FieldPowertrainTractionBatteryChargingIsCharging True if charging is ongoing. Charging is considered to be ongoing if energy is flowing from charger to vehicle.
 	FieldPowertrainTractionBatteryChargingIsCharging = "powertrainTractionBatteryChargingIsCharging"
+	// FieldPowertrainTractionBatteryCurrentPower Current electrical energy flowing in/out of battery. Positive = Energy flowing in to battery, e.g. during charging. Negative = Energy flowing out of battery, e.g. during driving.
+	FieldPowertrainTractionBatteryCurrentPower = "powertrainTractionBatteryCurrentPower"
```
Comment (Allyson-English) [inline pkg/vss/vehicle-structs.go:62]: do we want to take this opportunity to also populate `powertrainTractionBatteryChargingIsCharging` if we're seeing a positive value here?

Comment (elffjs) [reply pkg/vss/vehicle-structs.go:62]: I worry that cross-talk between the fields will not fit into the code generation.

Comment (KevinJoiner) [reply pkg/vss/vehicle-structs.go:62]: The codegen should handle multiple signals being populated from the same data point

**Review by KevinJoiner — COMMENTED** [review]
`make generate` will generate the code for this repo I can update the readme with make help.

File: pkg/vss/vehicle-structs.go (hunk 1 above)
Comment (elffjs) [reply pkg/vss/vehicle-structs.go:62]: Speaking lazily: we only get this for Tesla, and we're for sure populating `charging` for those.

**Pull request merged**